docs/concepts/tracking_costs_for_llm_providers.md

```diff
@@ -42,20 +42,21 @@ class BookRecommender(OpenAICall):
 
 
 stream = BookRecommender(genre="fantasy").stream()
-for chunk in stream:
+openai_stream = OpenAIStream(stream)
```

Comment:
we should at least mention async stream variants, right? Maybe just above when we introduce is? i.e. `BaseAsyncStream` -> `OpenAIAsyncStream`